shopfloor_base/utils.py

```diff
@@ -33,6 +34,37 @@ def wrapped(*args, **kwargs):
     return _ensure_model
 
 
+def catch_errors(response_error, exceptions_to_catch=None):
```

Comment:
```suggestion
def with_savepoint(response_error, exceptions_to_catch=None):
```
I would prefer this name, not blocking